src/pydrex/data/specs/spec.toml

```diff
@@ -79,15 +79,11 @@ log_level = "DEBUG"
 # DREX and simulation parameters are given in this section:
 [parameters]
 
-# Optional olivine volume fraction to use for multiphase simulations.
-# If enstatite_fraction is provided, they must sum to 1.
-# Otherwise, the remainder is assumed to be enstatite.
-# olivine_fraction = 1.
+# Optional mineral phase content to use for the aggregate.
+# phase_content = ["olivine", "enstatite"]
```

Comment:
'phase assemblage' rather than 'phase content'? I realise that `phase_content` is used everywhere, but I feel like `phase_assemblage = ["olivine", "enstatite"]` makes more sense. Up to you.

Reply:
Just a note that you have not changed it in the text, whether in comments or strings. From a quick glance, I would say you should replace `phase content` with `phase assemblage` and `phase contents` with `phases`. But there could be instances of `phase content` that read better as `phase`.

Reply:
Thanks, done.